docs/api/en/helpers/AxesHelper.html

```diff
@@ -25,9 +25,9 @@ <h2>Code Example</h2>
 		<h2>Examples</h2>
 
 		<p>
-			[example:webgl_geometries WebGL / geometries]<br/>
+			[example:webgl_buffergeometry_compression WebGL / buffergeometry / compression]<br/>
 			[example:webgl_geometry_convex WebGL / geometry / convex]<br/>
-			[example:webgl_geometry_spline_editor WebGL / geometry / spline / editor]
+			[example:webgl_loader_nrrd WebGL / loader / nrrd]<br/>
```

Comment:
There is no need for a `<br/>` at the last line.

Reply:
Updated.